packages/x-tailwindcss/src/x-tailwind-plugin/components/facet-filter/simple.ts

```diff
@@ -0,0 +1,32 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `simple` variant for component `facet-filter`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function facetFilterSimple(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    simple: {
+      '&:hover': {
+        color: theme('colors.neutral.90'),
```

Comment:
Currently every simple filter looks the same regardless if they have a color variant added. It is only when you select the filter then you can see something different. Is this behaviour intended?